docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+* enabling and encourage participants to solve problems in a way that builds confidence
+* supporting the project scientist in establishing workflows on GitHub
+* mentoring individual team members requesting additional support
+* working with the team to navigate technical or team dynamic challenges
+
+## Cloud computing resource organizers
+
+For many events we use centralized cloud computing resources, for example JupyterHub deployed on a commercial cloud platform. This requires someone who can deploy these resources and be available to assist with any troubleshooting during the event. 
+
+## overall coordination
+
+There may be one or several people considered overall leaders of the event. These are often the people who initiated the idea of the hack week in the first place. Within a more traditional organizational structure, there may be one person responsible for most of the decision making and ensuring everthing gets done, however this can place a significant burden on a single person. In a more distributed leadership model, multiple people have autonomous decision making power informed and supported by cross-team communications. 
+
+Responsibilities of overall coordinators include:
+
+* providing the impetus and energy for initiating and moving the event forward
+* calling meetings, establishing effective channels of communication and linking up the right people
+* identifying duplication of effort and increasing team efficiencies
+* acting as the main point of contact for outreach and communcations outside of the group
+* writing proposals and seeking funding
+* leading efforts to recruit members of the organizing committee
+* keeping an eye on the big picture to be sure everyone is on schedule
+* leading challenging discussions and mediating conflicts
+
+# Recruiting an organizing committee
+
+Depending on the size of the group who initiate the idea of a hack week, it may or may not be necessary to put time into recruiting others to assist as members of the organizing committee. In the case of an institution such as a university department starting a hack week, there may be enough local administrative support and tutorial leads without needing to find additional people. In other cases, for example our recent ICESat-2 hack week aimed at building software tools for a new NASA mission, we wanted to have participation from a broad community, especially members of the satellite's science team who had already begun to build software tools. In this case a broader range of recruitment efforts were necessary.
+
+Because the organizing committee has a responsibility to model the principles and norms of behavior we aim to promote at a hack week, it is important to find people whose values align with this mission. Similarly, we want the organizing committee to be as diverse as possible to maximize the number of different perspectives in the room, and to offer role models to an equally diverse range of hack week participants.  

```

Comment:
Have we written down somewhere what the "principles and norms of behaviour we aim to promote at a hack week" are? I don't think we've been super explicit about that either in the paper or here, so it's worth thinking about what those common principles and norms are, and add them somewhere? Maybe that's what you've been planning to propose for the objectives section?